applications/openshift/networking/default_ingress_ca_replaced/tests/ocp4/e2e-remediation.sh

```diff
@@ -1,4 +1,35 @@
 #!/bin/bash
 
-# we are using an existing default secret name for testing, so it won't cause any subsequent failures on testing routes. 
+# Reuse an existing certificate so we don't have to regenerate one.
+BUNDLE=$(oc get configmap -n openshift-apiserver trusted-ca-bundle -o json | jq '.data."ca-bundle.crt"')
+
+# Create it in the openshift-config namespace. If we don't do this, the
+# machineconfig cluster operator will fail to find it and eventually go into a
+# degraded state.
+cat << EOF | oc create -f -
+apiVersion: v1
+kind: ConfigMap
+data:
+  ca-bundle.crt: $BUNDLE
+metadata:
+  name: trusted-ca-bundle
+  namespace: openshift-config
+EOF
+
+cat << EOF | oc create -f -
+apiVersion: v1
+kind: ConfigMap
+data:
+  ca-bundle.crt: $BUNDLE
+metadata:
+  name: trusted-ca-bundle
+  namespace: openshift-config-managed
```

Comment:
Experimenting with this to see if it works around the following issue:

```
I0730 23:51:58.527582       1 log.go:198] Failed to sync additional trust bundle configmap openshift-config-managed/trusted-ca-bundle: failed to update trusted CA bundle configmap 'openshift-config-managed/trusted-ca-bundle': ConfigMap "trusted-ca-bundle" is invalid: []: Too long: must have at most 1048576 bytes
```

By creating it manually instead of relying on the sync logic in the networking operator.